tests/test_quick.py

```diff
@@ -0,0 +1,43 @@
+import pytest
+
+from ciphey import decrypt
+from ciphey.iface import Config
+from click.testing import CliRunner
+from ciphey.ciphey import main
+from ciphey.basemods.Checkers import human
+import mock

```

Comment:
New quick testing mode.

```
pytest -k quick
```

Tests Ciphey in under 10 seconds. Very useful for "Does this change break any major part of Ciphey?"